MarcusBarnes/mik
# Address #329.

**Github issue**: (#329)

# What does this Pull Request do?

As described in detail in #329, this PR applies the steps described in the "Refreshing a repository after changing line endings" section of https://help.github.com/articles/dealing-with-line-endings/.

# What's new?

Makes the following files fall under the line ending rules defined in .gitattributes:

* extras/scripts/postwritehooks/object_timer.php
* extras/sfu/mappings_files/icohc_mappings.csv
* src/metadatamanipulators/InsertXmlFromTemplate.php
* src/metadatamanipulators/NormalizeDate.php
* src/metadatamanipulators/SplitRepeatedValues.php
* src/metadatamanipulators/SplitRepeatedValuesInJson.php

# How should this be tested?

The problem described in #329 only manifests itself in Linux machines, as far as we know. To test this PR, do the following on a Linux machine:

1. clone the mik repo into a clean location (e.g., /tmp)
1. in the master branch, `cd mik`
1 `git status` should show that the files listed above are "modified"
1. check out the issue-329 branch
1. reruning `git status` should result in a clean working directory
1. to ensure that this change has no side effects, `cd` to the mik directory and run the following `phpunit --exclude-group inputvalidators --bootstrap vendor/autoload.php tests`

# Interested parties
@MarcusBarnes
